docs/installation/ha-installation.md

```diff
@@ -2,10 +2,58 @@
 title: High Availability Installation Overview
 ---
 
-This documentation explains High Availability architecture for Karmada deployment.
-The Karmada High Availability deployment is very similar to the Kubernetes High
-Availability Deployment. To deploy Karmada in a HA(High Availability) environment
+This documentation explains high availability for Karmada.
+The Karmada high availability architecture is very similar to the Kubernetes high
+availability architecture. To deploy Karmada in a HA (High Availability) environment
 we can create multiple Karmada API servers instead of using a single API server.
 
+In Karmada HA a member cluster can get registered to the multiple Karmada control planes.
+So even if a Karmada control plan goes down, you can still manage your clusters
+using the other Karmada control planes.
+
+Here is a diagram of Karmada HA architecture.
+
 ![Karmada HA 1](../resources/general/ha-karmada-1.png)
-![Karmada HA 2](../resources/general/ha-karmada-2.png)
\ No newline at end of file
+![Karmada HA 2](../resources/general/ha-karmada-2.png)
```

Comment:
These words and pictures seem a little bit confusing. I guess we can remove them as they are fully explained in the below content.

> In Karmada HA a member cluster can get registered to the multiple Karmada control planes.

The `register` in Karmada usually means the action a cluster joins to control plane, usually a cluster just needs to register to one control plane.

> ![Karmada HA 1](../resources/general/ha-karmada-1.png)
![Karmada HA 2](../resources/general/ha-karmada-2.png)

The two pictures show the 3 ETCDs are completely independent, which conflicts with the below `Stacked etcd topology`.